arrow-libs/core/arrow-core/src/commonMain/kotlin/arrow/core/continuations/Raise.kt

```diff
@@ -62,6 +64,47 @@ public interface Raise<in R> {
   
   /** Raise a _logical failure_ of type [R] */
   public fun raise(r: R): Nothing
+
+  /**
+   * Accumulate the errors obtained by executing the [block]
+   * over every element of [this] using the given [semigroup].
+   * The error type [E] of the [block] may be different from
+   * the error type [R] of the surrounding computation.
+   */
+  public fun <E, A, B> Iterable<A>.mapOrAccumulate(
+    embed: (E) -> R,
+    semigroup: Semigroup<@UnsafeVariance R>,
+    block: Raise<E>.(A) -> B
+  ): List<B> {
+    var error: R? = null
```

Comment:
Using `null` here is unsafe due to the nested null problem.

Reply:
I didn't know about `EmptyValue`, this has been fixed now.